sync.sh

```diff
@@ -1,22 +1,60 @@
-#!/bin/sh -ex
+#!/bin/bash -e
 
 REPO=../auto-opam-repository
 UPSTREAM=git://github.com/ocaml/opam-repository
 GEN=`pwd`/generate.native
 PULL=`pwd`/create_pull_request.native
-V=4.03.0
 if [ ! -d ${REPO} ]; then
   git clone git@github.com:bactrian/opam-repository ${REPO}
 fi
 
 BRANCH=sync-prs-`date +%s`
 HRDATE=`date +%c`
 cd $REPO
+# @@DRA Cleaner to ensure that origin/upstream is same as $UPSTREAM
+#       and do a fetch followed by hard reset to upstream
```

Comment:
I'm not a fan of git pull - doing a hard reset would also deal with the problem you discovered the other day where opam-repository/master appeared to have been force pushed. What do you think?